CRVResponse/src/CRVTest_module.cc

```diff
@@ -52,6 +52,7 @@ namespace mu2e
     art::InputTag _protonBunchTimeTag;
 
     TNtuple  *_recoPulses;
+    TNtuple  *_recoPulses2;
```

Comment:
Seeing how this module is evolving I suggest moving it to a new directory, possibly CRVCalib or CRVMonitoring.

Reply:
This module is not used by anyone else but me, and I have been using it to test various things and changed it whenever I needed to test new things. For simplicity, I added it to the repository.